test/JoeMakerV4.test.ts

```diff
@@ -206,41 +206,44 @@ describe("joeMakerV4", function () {
         value: ethers.utils.parseEther("1"),
       })
 
-      await this.joeMakerV4.convert(this.tractor.address, this.wavax.address)
+      await this.joeMakerV4.convert(this.tractor.address, this.wavax.address, "100")
 
       expect(await this.usdc.balanceOf(this.joeMakerV4.address)).to.equal(0)
       expect(await this.usdcDai.balanceOf(this.joeMakerV4.address)).to.equal(0)
       expect(await this.usdc.balanceOf(BAR_ADDRESS)).to.equal("186168809")
```

Comment:
Again, I think we need better assertions here; the expectation is that Bar receives a certain amount of JOE